src/main/resources/org/openstreetmap/atlas/sqlite/tables_v1.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE metadata (key text NOT NULL, value);
+CREATE TABLE dictionary (key integer PRIMARY KEY, value);
+CREATE TABLE node (id integer PRIMARY KEY, checksum text, shard text, geom blob NOT NULL);
```

Comment:
You might want to considering storing incoming and outgoing edge identifiers for a node either in an array column or in another table. Although if you store it in a separate table that other table could get very large very quickly.

Reply:
For rendering purposes, this can just go in the properties. If this becomes an underlying format for atlas, absolutely! There's a ton of things we can do to optimize the schema and the encoding. For one, the database gets big fast, and some of the compression techniques you see in the current atlas format are good. Like, the Mapquest LineString compression, and the lat lons packed into int64s. SQLite gives you the indexing, but how you store the underlying data being indexed is up to us.